### DISCUSSION TITLE
Auto-delete fork

### DISCUSSION THREAD

**Discussion by guerinoni** [community/community#9883] [Other Feature Feedback, Questions, & Ideas]
I would like to see a feature that automatically delete a fork if there isn't activity for a while… Maybe it can be a custom period of time, but many people fork just for a PR and let the repo still in personal page…
If there isn't a new branch or some real condition to stay forked, at least send a notification for remove it :)
Just for keep cleaning personal page.

**Comment by Rudxain** [comment]
I delete forks whenever ["Yours"](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/viewing-branches-in-your-repository) reaches 0. So, my GC-algorithm is just reference-counting, lol
